ui/components/multichain/network-list-item/network-list-item.js

```diff
@@ -93,6 +100,14 @@ export const NetworkListItem = ({
             name
           )}
         </Text>
+        {isDeprecatedNetwork ? (
+          <Tooltip
+            title={t('auroraDeprecationWarning', [AURORA_ETH_DISPLAY_NAME])}
```

Comment:
I think this is fine for now but we'll eventually want a better system for allowing more than one deprecated network.

Reply:
Noted! Yeah, so for the future we have plans to add a deprecated key in the networks list and update this file based on that. cc @kevinghim